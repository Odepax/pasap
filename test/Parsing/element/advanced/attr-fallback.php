<dl class="AttrFallback">
	<dt>One</dt> -- <dd><?= $this->attr('one', '1 (fallback)') ?></dd>
	<dt>Two</dt> -- <dd><?= $this->attr('two') ?></dd>
	<dt>Three</dt> -- <dd><?= $this->attr('three', '3 (fallback)') ?></dd>
</dl>
