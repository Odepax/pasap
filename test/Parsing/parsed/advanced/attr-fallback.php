<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<attr-fallback one="1" two="2" three="3"/>
		<attr-fallback one="1"/>
	</body>
</html>
