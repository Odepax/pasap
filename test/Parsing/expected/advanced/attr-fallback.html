<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<dl class="AttrFallback">
			<dt>One</dt> -- <dd>1</dd>
			<dt>Two</dt> -- <dd>2</dd>
			<dt>Three</dt> -- <dd>3</dd>
		</dl>
		<dl class="AttrFallback">
			<dt>One</dt> -- <dd>1</dd>
			<dt>Two</dt> -- <dd></dd>
			<dt>Three</dt> -- <dd>3 (fallback)</dd>
		</dl>
	</body>
</html>
